pubspec.lock

```diff
@@ -7,14 +7,14 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "49.0.0"
+    version: "50.0.0"
```

Comment:
nit: tangling deps updates with feature work is best avoided - it makes it harder to find the cause of problems if some of them cause regressions.